pkg/microservice/aslan/core/workflow/handler/policy.yaml

```diff
@@ -6,6 +6,12 @@ rules:
     alias: "查看工作流"
     description: ""
     rules:
+      - method: GET
+        endpoint: "api/aslan/workflow/workflow"
```

Comment:
Is there any difference between `/api` and `api` formats for permission judgment?

Reply:
no difference